<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin</class>
 <widget class="QMainWindow" name="MainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="2" column="0">
     <widget class="Line" name="line_bottom">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>2</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>2</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget_background" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <widget class="Line" name="line_central">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QStackedWidget" name="stackedWidget">
         <widget class="QWidget" name="page_record"/>
         <widget class="QWidget" name="page_classify"/>
         <widget class="QWidget" name="page_analysis"/>
         <widget class="QWidget" name="page_asset"/>
         <widget class="QWidget" name="page_book"/>
         <widget class="QWidget" name="page_setting"/>
         <widget class="QWidget" name="page_borrow"/>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QWidget" name="widget_dock" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="4" column="0">
           <widget class="QPushButton" name="btn_book">
            <property name="minimumSize">
             <size>
              <width>135</width>
              <height>55</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>135</width>
              <height>55</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>账本管理</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="btn_asset">
            <property name="minimumSize">
             <size>
              <width>135</width>
              <height>55</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>135</width>
              <height>55</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>资产管理</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="btn_record">
            <property name="minimumSize">
             <size>
              <width>135</width>
              <height>55</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>135</width>
              <height>55</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>记一笔账</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="btn_analysis">
            <property name="minimumSize">
             <size>
              <width>135</width>
              <height>55</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>135</width>
              <height>55</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>账目分析</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="btn_borrow">
            <property name="minimumSize">
             <size>
              <width>135</width>
              <height>55</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>135</width>
              <height>55</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>借还记录</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QPushButton" name="btn_setting">
            <property name="minimumSize">
             <size>
              <width>135</width>
              <height>55</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>135</width>
              <height>55</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>软件设置</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="btn_classify">
            <property name="minimumSize">
             <size>
              <width>135</width>
              <height>55</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>135</width>
              <height>55</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>分类管理</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="Line" name="line_top">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>2</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>2</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
